### PR TITLE
Added ESLint linebreak issue

### PR DESCRIPTION
**Pull request by Heyjp**
Just updated README with your proposed link to fix the ESLint issue on Windows computers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ Our app helps (aspiring) developers to prepare for job interviews with questions
 
 ## Install and Build
 
+### Developer Issues
```
Comment (zsoltime) [inline README.md]: It should be an `h4` tag so there should be four pound-signs/hashtags. Also "issues" should not be capitalised to match other `h4`s.

File: README.md [hunk 2]
```diff
@@ -63,6 +63,12 @@ Our app helps (aspiring) developers to prepare for job interviews with questions
...
+
+If running on Windows whilst building the app you might encounter a linebreak error message from ESLint.
+
+To fix simply add the to bash the [following](https://gist.github.com/RadValentin/dd8b9d70448413301e9a274ed54661a5).
```
Comment (zsoltime) [inline README.md]: There is a typo there.

**Review by zsoltime — CHANGES_REQUESTED** [review]
Could you check my comments and fix these issues? Thanks 😃

**Comment by Heyjp** [comment]
Changed to h4, also couldn't see typo you were referring to though.

**Review by zsoltime — APPROVED** [review]
Great, thanks. 👍

**Pull request merged**